packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -168,7 +171,8 @@ class IconButton extends StatelessWidget {
     return new InkResponse(
       onTap: onPressed,
       child: result,
-      radius: math.max(size, Material.defaultSplashRadius),
+      radius: math.max(48.0, iconSize + math.min(padding.horizontal, padding.vertical))
+          * 0.55, // x 0.5 for diameter -> radius and + 10% overflow so it doesn't look too square.
```

Comment:
I'm unsure about this change.  Is there anything in the spec that says what these values should be?  I would expect in the default (48x48) case, that Material.defaultSplashRadius is the proper radius.  I think we got that by measuring something in the spec, but I'm not certain.

Your calculation looks reasonable for the non-default case.  I just wonder if we should use Material.defaultSplashRadius rather than using 48.0 * 0.55 as the left parameter to the max.